python/tank/deploy/tank_commands/install.py

```diff
@@ -358,12 +360,13 @@ def run_interactive(self, log, args):
         :param args: command line args
         """
 
-        if "--preserve-yaml" in args:
-            preserve_yaml = True
-            args.remove("--preserve-yaml")
-            log.info("Note: Using yaml parser which preserves structure and comments.")
+        # look for an --use-pyyaml flag
+        if constants.LEGACY_YAML_PARSER_FLAG in args:
+            preserve_yaml = False
+            args.remove(constants.LEGACY_YAML_PARSER_FLAG)
+            log.info("Note: Falling back on pre-0.18 legacy yaml parser.")
```

Comment:
pre-0.18 yaml parser is confusing since that version is not even out yet. We should stick to 0.17.whatever.